alcs-frontend/src/app/shared/inline-editors/inline-text/inline-text.component.ts

```diff
@@ -41,14 +42,16 @@ export class InlineTextComponent implements AfterContentChecked {
   confirmEdit() {
     if (this.pendingValue !== this.value) {
       this.save.emit(this.pendingValue?.toString() ?? null);
-      this.value = this.pendingValue;
+
+      if (this.updateOnSave) {
+        this.value = this.pendingValue;
+      }
     }
 
     this.isEditing = false;
   }
 
   cancelEdit() {
     this.isEditing = false;
-    this.pendingValue = this.value;
```

Comment:
This may cause other issues, if I edit an inline text field and click cancel then click edit again, does it show the original value or the one I had cancelled previously?

Reply:
This line doesn't do anything, because `pendingValue` is already set, and `value` hasn't changed.

Reply:
`pendingValue` changes with the input, but it'll be reset by `startEdit` when you edit again.